_includes/manuals/1.24/3.2.4_tuning_options.md

```diff
@@ -0,0 +1,54 @@
+The Foreman installer supports automatic tuning of your environment using predefined tuning profiles.  These tuning profiles are the result of a culmination of extensive learning from foreman environments deployed at scale in large customer environments.
+
+When the foreman-installer is run, it is deployed with a `default` predefined tuning profile. The definition of this profile can be found in `/usr/share/foreman-installer/config/foreman.hiera/tuning/common.yaml`.
+
+Other than the default tuned profile, foreman-installer supports 4 different tuning profiles:
+
+  * medium
+  * large
+  * extra-large
+  * extra-extra-large
+
+Definitions of various tuning profiles can be found in this directory `/usr/share/foreman-installer/config/foreman.hiera/tuning/sizes/`.  Based on your environment needs, use one of the tuning profiles in the installer as shown below.
+
+```bash
+foreman-installer --tuning medium
+```
+
+```bash
+foreman-installer --tuning large
+```
+
+```bash
+foreman-installer --tuning extra-large
+```
+
+```bash
+foreman-installer --tuning extra-extra-large
+```
+
+To reset to the default profile:
+
+```bash
+foreman-installer --tuning default
+```
+
+**Note** Using the `--tuning` option above does not update `/ect/foreman-installer/custom-hiera.yml`, instead it directly updates the required configuration as specified in the corresponding tuning profile.
```

Comment:
I think we can leave this out. This was part of the initial design but was never communicated. Adding it here will only confuse people. (There's also a typo in `/etc`)

Reply:
Many users are used to seeing override in custom-hiera.yml for tuning profile. Hence I added it. This was also brought out by [QE](https://bugzilla.redhat.com/show_bug.cgi?id=1772390#c0). In my new commit, I added a new that the users can still use custom-hiera.yml to override any configuration. Let me know what you think.

Reply:
Good point. It may also be good to recommend users who have previously tuned manually should review their values. They can overwrite built in tuning but is not recommended.